### ISSUE TITLE
APT source whitelist request for ppa:zoogie/sdl2-snapshots

### ISSUE THREAD

**Issue by BanzaiMan** [travis-ci/apt-source-whitelist#45]
This replaces travis-ci/travis-ci#4263.

The original text by @startaq follows

------

Contains SDL2 libraries.

https://launchpad.net/~zoogie/+archive/ubuntu/sdl2-snapshots

**Comment by endrift** [comment]
+1, I need this before I can migrate https://github.com/mgba-emu/mgba to the new system.

**Comment by DrMcCoy** [comment]
Yeah, I still need that too. While https://github.com/travis-ci/apt-package-whitelist/pull/450 added the sdl2 packages, it seems the test thing pulled it out of the debian-sid source?

Using the debian-sid source is not really viable for me, since I also need Boost 1.55, and the debian-sid source seems to take precedence over the boost-latest source.

And using Boost 1.55 out of debian-sid doesn't work due to it also needing new libc-bin, libc6-dev and locales packages, and then it abort when locales tries to replace a file that's also in the old Ubuntu libc-bin package. I.e. that's an unholy dependency mess. See [this build log](https://api.travis-ci.org/jobs/73290927/log.txt?deansi=true), if it's of any interest.

I really wonder about the usability of the Debian Sid repository on Ubuntu Precise in general. Given the huge time difference between the two, and Debian Sid being called "unstable" for a reason, that just smells dangerous from every single angle. Especially for C++, now that Debian is starting to roll out gcc 5.1. and 5.2, with the new ABI.

**Comment by LordAro** [comment]
Any update on getting this added? I know debian-sid adds the packages, but it seems they don't work well with 12.04 (funny that)

**Comment by SirVer** [comment]
+1. this would be really nice to have.

**Comment by dessant** [comment]
The ppa does not seem to be maintained anymore, it is two versions behind.
